routes/callback.ts

```diff
@@ -0,0 +1,31 @@
+import express from 'express'
+import { sgidService } from '../services/sgid-client.service'
+import { formatData } from '../utils'
+import { RANDOMNONCE } from '../config'
+import { nodeCache } from '../services/node-cache.service'
+
+/**
+ * Main controller function to generate the callback page
+ */
+export const callback = async (req: express.Request, res: express.Response) => {
+  try {
+    const { code, state } = req.query
+    const codeVerifier = nodeCache.get(req.cookies.sessionId)
```

Comment:
minor suggestion to keep the cookie name somewhere as a constant so it's easy to find

Reply:
Resolved in [6a21f7a](https://github.com/opengovsg/sgid-test-app/pull/94/commits/6a21f7abfb16f44d9ca5bc9fcf4c6c4e26fd6910).